src/client/store/ethereum/actions.js

```diff
@@ -33,21 +33,12 @@ export function getTransactionId(txMethod, params) {
 
 export function initializeProvider() {
   return async (dispatch) => {
-    const provider = await detectMetaMask();
+    let provider = await detectMetaMask();
 
     if (provider) {
       provider.on('accountsChanged', async (accounts) => {
         dispatch(await handleAccountChange(accounts));
       });
-
-      provider
-        .request({ method: 'eth_accounts' })
-        .then(async (accounts) => {
-          dispatch(await handleAccountChange(accounts));
-        })
-        .catch(() => {
-          // Do nothing ..
-        });
```

Comment:
Removing this will cause the user to press "enable" manually which is fine for me, just noting this here.

Reply:
Yes! it does. I removed it because I was having a bug where, once metamask has already been enabled/approved for our site, if the page was refreshed, it wouldn't detect metamask at all